src/main/java/org/veupathdb/lib/gradle/container/tasks/jaxrs/JaxRSPatchDates.kt

```diff
@@ -0,0 +1,47 @@
+package org.veupathdb.lib.gradle.container.tasks.jaxrs
+
+import org.veupathdb.lib.gradle.container.tasks.base.JaxRSSourceAction
+import java.io.File
+import java.util.Arrays
+import java.util.stream.Stream
+
+open class JaxRSPatchDates : JaxRSSourceAction() {
+
+  companion object {
+    const val TaskName = "jaxrs-patch-dates"
+
+    private val replacements = listOf(
+      "(Date " to "(OffsetDateTime ",
+      " Date " to " OffsetDateTime ",
+      "import java.util.Date;" to "import java.time.OffsetDateTime;",
+      "  @JsonDeserialize(\n      using = TimestampDeserializer.class\n  )\n" to "",
```

Comment:
How does this line work?  Looks like TimestampDeserializer would have to be imported in the generated class files, but I don't see you adding it and it doesn't seem like it would be a generated class.  I assume it's Jackson but then would still need to be imported, right?

Reply:
Jackson knows how to handle `java.util.Date` and `java.time.*` types by itself.  This `@JsonDeserialize` annotation was here just to provide the RFC3339 format for deserialization via one of Jackson's built-in date formats.

```java
public class TimestampDeserializer extends StdDeserializer<Date> {
  private static final StdDateFormat DATE_PARSER = new StdDateFormat();

  public TimestampDeserializer() {
    super(Date.class);}

  public Date deserialize(JsonParser jsonParser, DeserializationContext jsonContext) throws
      IOException, JsonProcessingException {
    try {
      ObjectMapper mapper  = new ObjectMapper();
      String dateString = mapper.readValue(jsonParser, String.class);
      Date date = DATE_PARSER.parse(dateString);
      return date;
      } catch (ParseException e) {throw new IOException(e);
    }
  }
}
```